src/Cli/Commands/InitOptions.cs

```diff
@@ -36,7 +36,8 @@ public class InitOptions : Options
             bool graphqlDisabled = false,
             CliBool restEnabled = CliBool.None,
             CliBool graphqlEnabled = CliBool.None,
-            CliBoolean restRequestBodyStrict = CliBoolean.None,
+            CliBool restRequestBodyStrict = CliBool.None,
+            CliBool nestedInsertOperationEnabled = CliBool.None,
```

Comment:
To track as a separate issue, I think we should add a different `dab` command that helps to enable or disable a feature, should not be only possible through the `init` command - which is the case today.

Reply:
I've created issue [#2000 ](https://github.com/Azure/data-api-builder/issues/2000) for tracking this.